templates/group.j2

```diff
@@ -1,8 +1,6 @@
 {% extends 'base.j2' %}
 
 {% block main %}
-  <h2 data-sassdoc="page-title">{{ pageTitle }}</h2>
```

Comment:
Hm... On smaller sizes (with the menu collapsed), this feels like useful context to me.